qa/cephfs/mount/kclient/overrides/distro/rhel/rhel_7.yaml

```diff
@@ -1 +0,0 @@
-.qa/distros/all/rhel_7.yaml
```

Comment:
This link needs to be changed to `.qa/distros/all/rhel_8.yaml`.

Reply:
Fixed.